package.json

```diff
@@ -8,12 +8,12 @@
     "test": "eslint ."
   },
   "devDependencies": {
-    "eslint": "^2.0.0",
-    "eslint-plugin-react": "^4.0.0"
+    "eslint": "^3.1.1",
+    "eslint-plugin-react": "^6.0.0-alpha.1"
```

Comment:
Do you expect this to hit final before we do a release of v3?  Otherwise I would go with v5.